tests/test_path_tar.py

```diff
@@ -1,13 +1,48 @@
 # built-in
 from pathlib import Path
 
+# external
+import pytest
+
 # project
 from dephell_archive import ArchivePath
 
 
 sdist_path = Path(__file__).parent / 'requirements' / 'sdist.tar.gz'
 
 
+def test_toplevel(tmpdir):
```

Comment:
happy to duplicate these over in test_path_zip, or create a new test file `test_path_meta.py`/`test_path_meta.py` for tests which do not depend on the tar/zip implementation.